trace/opencensus/trace/ext/django/config.py

```diff
@@ -74,6 +81,20 @@ def __getattr__(self, attr):
         return module_class
 
 
+def _set_default_configs(user_settings, default):
+    """Set the default value to user settings if user not specified
+    the value.
+    """
+    config_keys = user_settings.keys()
+    supported_keys = default.keys()
+
+    for key in supported_keys:
+        if key not in config_keys:
```

Comment:
This could probably be `if key not in user_settings` to check if the key exists rather than checking if the value is found in the list of keys.

Reply:
Done. Thanks!